recipes/jquery-mobile/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "jquery-mobile" %}
+{% set version = "1.4.5" %}
+{% set sha256 = "b160c44ead40944995452742e6544b254596126b8f50e2e79a14d4e3e0c6ce37" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jquery/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  
+test:
+  commands:
+    - npm list -g jquery-mobile
+  requires:
+    - nodejs
+   
+about:
+  home: https://github.com/jquery/jquery-mobile
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: 'jQuery Mobile Framework'
+  description: |
+    jQuery Mobile is a unified, HTML5-based user interface system for all popular 
+    mobile device platforms, built on the rock-solid jQuery and jQuery UI foundation. 
+    Its lightweight code is built with progressive enhancement, and has a flexible, 
+    easily themeable design.
+  doc_url: https://jquerymobile.com/
+  dev_url: https://github.com/jquery/jquery-mobile
+
+extra:
+  recipe-maintainers:
+    - cshaley
```

Comment:
@cshaley -- can you please confirm that you want to be listed as a maintainer here?

Reply:
yes, and any packages @sannykr adds me to